CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Standard services may be enabled using specific `ejb_app_services.toml` file.
+  It supports only `configuration` and `btc-anchoring` services at the moment.
+
+  To enable services place `ejb_app_services.toml` file in the directory 
```

Comment:
with..with tautology. I propose to change this sentence to `... put `ejb_app_services.toml` file in EJB App's directory with the following content:`